server/motr_delete_key_value_action.cc

```diff
@@ -70,7 +70,7 @@ void MotrDeleteKeyValueAction::delete_key_value() {
           request, s3_motr_api);
     }
     motr_kv_writer->delete_keyval(
-        index_id, request->get_key_name(),
+        {index_id}, request->get_key_name(),
```

Comment:
whats the need of {...}

Reply:
It's because the signature of delete_key_value() was changed